data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -35,6 +33,9 @@
 import cdr_cleaner.cleaning_rules.valid_death_dates as valid_death_dates
 import cdr_cleaner.manual_cleaning_rules.negative_ppi as negative_ppi
 import cdr_cleaner.manual_cleaning_rules.ppi_drop_duplicate_responses as ppi_drop_duplicates
+import cdr_cleaner.manual_cleaning_rules.remove_operational_pii_fields as operational_pii_fields
+import sandbox
+from constants.cdr_cleaner.clean_cdr import DataStage as stage
```

Comment:
I know this isn't part of this code review, but why did we create a class, `DataStage`, to represent a list of constant values?

Reply:
I am not sure, I created data_stage in the Parameter to make clean_cdr run on particular stage. But Datastage is added by Chao, i'll find out.